modules/ecs6-class/line.js

```diff
@@ -2,6 +2,11 @@ const Point = require("./point");
 
 class Line {
     constructor({ point1 = new Point(), point2 = new Point(), n = undefined, slope = undefined }) {
```

Comment:
What happens when point1 is a string when n is a string ?? think about more pitfalls